docs/intro.md

```diff
@@ -5,7 +5,9 @@ sidebar_label: Overview
 ---
 # Overview of Komiser Documentation
 
-![Tailwarden](/img/docs-banner-tw.png)
+
+![Tailwarden](/img/komiser-overview-light.png#gh-light-mode-only)![Tailwarden](/img/komiser-overview-dark.png#gh-dark-mode-only)
```

Comment:
is it possible to use if else conditions here to show an image only if dark or light mode is enabled. If not, I would just keep the light version only and remove the dark mode version

Reply:
Opps, I added back the css that automatically detects the mode we are in, so without requiring an if else statement, we will only we showing one image at a time now.